R/dist-normal.R

```diff
@@ -1,30 +1,24 @@
 drawMusAndSigmas <- function(data,
-                             m0,
-                             k0,
-                             s_sq0,
-                             v0,
+                             mu,
+                             lambda,
+                             alpha,
+                             beta,
                              n_samples) {
 
-  N <- length(data)
-  the_mean <- mean(data)
+  n <- length(data)
+  xbar <- mean(data)
 
-  ssd <- sum((data - the_mean) ^ 2)
+  ss <- var(data) * n
```

Comment:
Sum of squared deviations is equivalent to variance multiplied by (n-1)

Reply:
good call